main.js

```diff
@@ -0,0 +1,146 @@
+/*
+ Bài 1: Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+*/
+
+const getArrayString = (string, lenght) => {
+  const arr = []
+  for (let i = 0; i < lengthArr; i++) {
+    arr.push(string)
+  }
+  return arr
+}
+
+/*
+ Bài 2: Viết 1 hàm để đảo ngược 1 mảng, không được dùng hàm array.reverse()
+*/
+
+const reverseArray = (arr = []) => {
+  const reversedArr = []
+  arr.forEach(item => {
+    reversedArr.unshift(item)
+  })
+  return reversedArr
+}
+
+/*
+ Bài 3: Xóa đi các giá trị được xem là sai
+*/
+
+const eraseFalsy = arr => arr.filter(element => element)
+
+/*
+ Bài 4: Tạo ra 1 array gồm các object có key và value tương ứng cặp array ban đầu
+*/
+
+const createObject = (arrayDatas = []) => {
+  const obj = {}
+  dataArr.forEach(arr => {
+    obj[arr[0]] = arr[1]
+  })
+  return obj
+}
+
+/*
+ Bài 5: Sắp xếp mảng tăng dần
+*/
+
+const sortArray = numbers => numbers.sort()
+
+/*
+ Bài 6: Kiểm tra input đầu vào có phải là object hay không?
+ */
+
+const checkTypes = obj => {
+  if (Array.isArray(obj) || typeof obj !== 'object') {
+    return false
+  } else {
+    return true
+  }
+}
+
+/*
+ Bài 7: Viết 1 hàm trả về các key (của 1 object) khác các key truyền vào
+*/
+
+const getObjectKeys = (obj, keys) => {
+  keys.forEach(key => {
+    for (let keyItem in obj) {
+      if (key === keyItem) {
+        delete (obj[key])
+      }
+    }
+  })
+  return obj
+}
+
+/*
+Bài 8: Viết hàm nhập vào 1 array có nhiều hơn 5 phần tử
+  Xóa phần tử số 2, 3 trong array Return mảng sau khi đã xóa
+*/
+const deleteArray = (array = []) => {
+  if (array.lenght < 5) {
+    return console.log("Array is more 5 elements")
+  }
+
+  const eleDelete = array.splice(1, 2)
+  return array
```

Comment:
```suggestion
  return array.splice(1, 2)
```

Khai báo `eleDelete` để làm gì khi ko dùng?